bitstore/controllers.py

```diff
@@ -33,7 +33,7 @@ def authorize(auth_token, req_payload):
     try:
         # Get request payload
         owner = req_payload.get('metadata', {}).get('owner')
-        dataset_name = req_payload.get('metadata', {}).get('name')
+        dataset_name = req_payload.get('metadata', {}).get('dataset', '')
```

Comment:
Yes it will 😯 - but isn't that what it should be if you forget to set dataset and it is expected (or you should get an exception!)